app/renderer/components/common/dropdown.js

```diff
@@ -37,6 +38,12 @@ class SettingDropdown extends ImmutableComponent {
   }
 }
 
+class BraveryPanelDropdown extends ImmutableComponent {
+  render () {
+    return <FormDropdown data-isBraveryPanel='true' {...this.props} />
```

Comment:
I think if you just let `data-isBraveryPanel` works. Above you're passing `true` as a string, which would evaluate prop to true anyway, so would any other string. For example, `data-isBraveryPanel='brave'` would be true as well.

You can double-check that if you pass `data-isBraveryPanel={true}`which will evaluate to boolean. Standardjs will warn you saying that it is not necessary:

```
Value must be omitted for boolean attributes
```

I would just `<FormDropdown data-isBraveryPanel {...this.props} />`.